cypress/support/pages/topics/pageTopics.js

```diff
@@ -214,21 +213,24 @@ class Topics {
     })
   }
 
-  enterLinkforElementTask(taskId) {
-    const env = Cypress.env()
-    const taskURL = env['BRB'] + '/homework/' + taskId
+  enterLinkforElementTask (taskId) {
+    const env = Cypress.env('BRB').toLowerCase()
+    const taskURL = `${env}homework/${taskId}`
```

Comment:
Please don't remove the "/" here (before "homework".
The env-url has to be written without the latest "/" in the local.env.json
Then it works. Otherwise it's failing.

Or is there a reason to change this? Then you should inform us, that we have to set the env-url with "/" (like we did it in the past), but also we have to check, if then all other tests are passing.

Reply:
Done.